gdal/swig/include/python/gdal_python.i

```diff
@@ -1356,6 +1358,19 @@ def WarpOptions(options=None, format=None,
             new_options += ['-cvmd', str(metadataConflictValue)]
         if setColorInterpretation:
             new_options += ['-setci']
+        if overviewLevel is not None:
+            if _is_str_or_unicode(overviewLevel):
+                overviewLevelStr = overviewLevel
+            else:
+                overviewLevelStr = ''
+                if overviewLevel is ...:
```

Comment:
What is this Python syntax (pardon my ignorance) ? Doesn't seem to work with python 2.7

Reply:
ellipsis (...) is a builtin constant:
https://docs.python.org/3/library/constants.html
I used it here to express a default value. 
I'm not sure this convention is widespread so maybe We'll use an empty string or NaN instead (or just remove this if statement and use only "AUTO" as is?

Reply:
I've forced push a fix:
Changed the default to be 'AUTO' instead of None.
removed the use of ellipsis